src/global_logic.js

```diff
@@ -2084,6 +2109,27 @@ module.exports.addSkilldataToTotals = function (totals, comb, arml, buff) {
         if (totals[key]["bahaAT"] > 50) totals[key]["bahaAT"] = 50;
         if (totals[key]["bahaHP"] > 50) totals[key]["bahaHP"] = 50
     }
+
+    // new epic 2nd skills
+    Object.values(totals).forEach(chara => {
+        if (numGrandEpic > 0) {
+            chara["ex"] += Math.min(LIMIT.grandEpic, numGrandEpic * countEpic * 4.0);
+        }
+        if (numResonanceStaff > 0) {
+            chara["normalOtherSante"] = Math.min(50.0, countWand * 1.0);
```

Comment:
This Math.min check can be removed ?
because staff x 10 does not reach that value
and normal TA has the limit 50% check.

we can check Math.min(10, countWand) to show explicitly the max amount here.

Reply:
Would add a countwand check so they don't go over 10 by accident and would add a 10.0 check incase of javascript rounding errors. 50.0 check is not needed, 10.0 is enuf because can't stack.

Reply:
ah, and found a bug, `=` should be `+=`
the epic code is the bottom of skill logic so always override "normalOtherSante"

```js
chara["normalOtherSante"] += Math.min(10, countWand) * 1.0;
```

- 10 ... chance to be float? it's counting number of wands then integer for Math.min
- 1.0 show the DA/TA value explicitly. (not effect to calc result, just code intent it's 1%/1 wand)